src/subscription/details/content/SubscriptionContent.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { Badge } from '@edx/paragon';
+
+const SubscriptionContent = ({ children, details }) => (
+  <div className="basket-section">
+    <h5 aria-level="2">
+      <FormattedMessage
+        id="subscription.purchase.details.heading"
+        defaultMessage="In your cart"
+        description="Describe what's in your details"
```

Comment:
I find this description a little confusing, perhaps something similar to the CartContent component would be clear? 
`Heading of the cart in the subscription details section`

Reply:
Updated.